pika/connection.py

```diff
@@ -319,6 +319,7 @@ class ConnectionParameters(Parameters):
     :param int|float socket_timeout: Use for high latency networks
     :param str locale: Set the locale value
     :param bool backpressure_detection: Toggle backpressure detection
+    :param json object client_props: setup your client properties (and overrides the default ones)
```

Comment:
Couple of problems here:

1. in python, it's a "dict", not "json object"
2. "json object" is two words; only one word types are allowed here per rst syntax.